provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/samconfig.toml

```diff
@@ -1,4 +1,4 @@
 version=0.1
 [default.build.parameters]
-build_image = "al2-graalvm:gradle"
+build_image = ["al2-graalvm:gradle"]
```

Comment:
note: `--build-sam` is multiple as defined [here](https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/build/command.py#L108-L122). If `build_image` is provided as a string in `samconfig.toml`, `click` will parse it as a tuple and break down the string. e.g.

```
2022-06-30 14:54:50,806 | Expand command line arguments to:
2022-06-30 14:54:50,807 | --template_file=mypath/202206/java11-graalvm-gradle/template.yaml --use_container --build_image=('a', 'l', '2', '-', 'g', 'r', 'a', 'a', 'l', 'v', 'm', ':', 'g', 'r', 'a', 'd', 'l', 'e') --build_dir=.aws-
```

Reply:
We can consider fixing how SAM CLI reads and parse `samconfig.toml`. But it seems it's quickest to just fix `samconfig.toml` here